lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -210,6 +217,11 @@ module Resources
       'start': '2020-07-10T13:00:00.000Z',
       'end': '2020-07-10T13:15:00.000Z',
       'minutesDuration': 15,
+      'slot': [
+        {
+          'reference': 'Slot/21265426-633867-6828001-60'
```

Comment:
Is 21265426-633867-6828001-60 a valid slot id for C1941?

Reply:
I think it is a C1941 ID since this is currently being documented for the create body field. Also, we are no longer required to use the C-1941 domain id's.

Reply:
According to this Teams conversation, real IDs are not required. https://teams.microsoft.com/l/message/19:e4575261adf84fabaa18f360d631ace7@thread.skype/1615582540081?tenantId=fbc493a8-0d24-4454-a815-f4ca58e8c09d&groupId=dce636bf-7e25-47d2-a255-c5e7f16951da&parentMessageId=1615582118117&teamName=SMART%2FFHIR&channelName=Resource%20Development&createdTime=1615582540081